docs/source/conf.py

```diff
@@ -14,3 +14,4 @@
 nb_render_image_options = {"width": "450px", "align": "center"}
 nb_execution_timeout = 600
 html_sidebars = {"**": []}
+jupyter_execute_notebooks = "cache"
```

Comment:
**[Question, non-blocking]** Can you point me to docs for this config options? I assumed they would be in the `jupyter-sphinx` docs but [I couldn't find anything there](https://jupyter-sphinx.readthedocs.io/en/latest/options.html#configuration-options).

Reply:
It's part of the Myst NB [config](https://myst-nb.readthedocs.io/en/v0.10.1/use/execute.html).